packages/calcite-components/src/utils/loadable.ts

```diff
@@ -111,3 +113,26 @@ export function setComponentLoaded(component: LoadableComponent): void {
 export function componentLoaded(component: LoadableComponent): Promise<void> {
   return promiseMap.get(component);
 }
+
+/**
+ * This helper util can be used to ensure a component is ready to be focused (The "componentDidLoad" stencil lifecycle method has been called and any internal elements are ready to be focused).
```

Comment:
Nitpick: `stencil` => `Stencil`

Reply:
Could you rephrase this part to state that this helps focusing by ensuring the component is loaded and rendered by the browser?